bin/README.md

```diff
@@ -1,22 +1,25 @@
 ## Running the web tool on transitionmonitor's docker-container
 
-Import the base images with:
-
-```bash
-docker image import https://tool.transitionmonitor.com/downloads/2diirunner_1.3.0.tar.xz 2diirunner:1.3.0
-```
-
 Ensure your working directory is PACTA_analysis, then run one of these workflows:
 
 * Run the web tool in a single step
 
 ```bash
+# Use the default portfolio name: TestPortfolio_Input
 ./bin/run-web-tool
```

Comment:
This is the easiest way to run all scripts in the web tool. Maybe the name `run-web-tool` is missleading.  The most common use I expect is testing, so maybe we should rename to `test-web-tool` or `test-r-scripts`?

Reply:
I think the reference to "web tool" is important here... there are many R script in this repo, some of which are never touched when running all there of the web tool scripts